src/components/ebay-snackbar-dialog/examples/02-action/template.marko

```diff
@@ -0,0 +1,20 @@
+<ebay-snackbar-dialog type="action" open=state.open on-close('closeSnackbar')>
+    This is the action snackbar
+</ebay-snackbar-dialog>
+<ebay-button on-click("openSnackbar")>
+    Open Action Snackbar
```

Comment:
The action button should be passed in by the app developer as part of the body. The same way that toast-dialog does it.

Reply:
that's right. fix in progress